canvas_account_admin_tools/requirements/base.txt

```diff
@@ -8,8 +8,8 @@ django-cached-authentication-middleware==0.2.1
 django-proxy==1.0.2
 django-redis-cache==1.6.5
 git+ssh://git@github.com/Harvard-University-iCommons/django-auth-lti.git@v1.2.5#egg=django-auth-lti==1.2.5
-git+ssh://git@github.com/Harvard-University-iCommons/django-icommons-common.git@v1.10.19#egg=django-icommons-common==1.10.19
-git+ssh://git@github.com/Harvard-University-iCommons/django-icommons-ui.git@v1.2.0#egg=django-icommons-ui==1.2.0
+git+ssh://git@github.com/Harvard-University-iCommons/django-icommons-common.git@v1.13#egg=django-icommons-common==1.13
+git+ssh://git@github.com/Harvard-University-iCommons/django-icommons-ui.git@v1.2.1#egg=django-icommons-ui==1.2.1
```

Comment:
Note to make this work at the moment you'll have to pull Harvard-University-iCommons/django-icommons-common#157 and https://github.com/Harvard-University-iCommons/django-icommons-ui/pull/82 locally (or reference the branches from github), as the tags haven't been created yet.